codecov.yml

```diff
@@ -0,0 +1,3 @@
+ignore:
+  - napari/_version.py
+  - "**/tests/*.py"
```

Comment:
I noticed our code coverage is approximately 0 and wondered if this might be the culprit.

Reply:
We don't have a similar line in scikit-image: https://github.com/scikit-image/scikit-image/blob/master/.codecov.yml